java/AvroOneTopicManySubjects/flink-consumer/src/main/java/com/amazonaws/services/msf/StreamingJob.java

```diff
@@ -0,0 +1,130 @@
+package com.amazonaws.services.msf;
+
+import com.amazonaws.services.kinesisanalytics.runtime.KinesisAnalyticsRuntime;
+import com.amazonaws.services.msf.avro.AirQuality;
+import com.amazonaws.services.msf.avro.RoomTemperature;
+import org.apache.flink.api.common.eventtime.WatermarkStrategy;
+import org.apache.flink.api.common.functions.FilterFunction;
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.java.functions.KeySelector;
+import org.apache.flink.connector.kafka.source.KafkaSource;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.LocalStreamEnvironment;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.kafka.clients.admin.*;
+import org.apache.kafka.common.errors.TopicExistsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Sample Flink application that can run in Amazon Managed Service for Apache Flink
+ * <p>
+ * It demonstrates how to process records in a Kafka topic with varying schema when
+ * they are produced using RecordName subject name strategy in Confluent Schema Registry.
+ */
+public class StreamingJob {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StreamingJob.class);
+    // Name of the local JSON resource with the application properties in the same format as they are received from the MSF runtime
+    private static final String LOCAL_APPLICATION_PROPERTIES_RESOURCE = "flink-consumer-dev.json";
+
+    // Names of the configuration group containing the application properties
+    private static final String APPLICATION_CONFIG_GROUP = "FlinkApplicationProperties";
+
+    public static void main(String[] args) throws Exception {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+        if (isLocal(env)) {
+            env = StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(new org.apache.flink.configuration.Configuration());
```

Comment:
Creating the local env with webUI is no longer required. With 1.20 the UI is automatically created when running locally if the flink-web dependency is included